tests/models/test_modeling_common.py

```diff
@@ -1433,6 +1434,52 @@ def get_memory_usage(storage_dtype, compute_dtype):
             or abs(fp8_e4m3_fp32_max_memory - fp32_max_memory) < MB_TOLERANCE
         )
 
+    @require_torch_gpu
+    def test_group_offloading(self):
+        init_dict, inputs_dict = self.prepare_init_args_and_inputs_for_common()
+        torch.manual_seed(0)
+
+        def run_forward(model):
```

Comment:
We should add a memory check here as well against a forward pass that's fully on GPU.